ons_alpha/bundles/models.py

```diff
@@ -6,33 +6,26 @@
 from django.utils.timezone import now
 from modelcluster.fields import ParentalKey
 from modelcluster.models import ClusterableModel
-from wagtail.admin.panels import FieldPanel, FieldRowPanel, InlinePanel, PageChooserPanel
+from wagtail.admin.panels import FieldPanel, FieldRowPanel, HelpPanel, InlinePanel
 from wagtail.fields import StreamField
 from wagtail.models import Orderable, Page
 from wagtail.search import index
 
-from ..datasets.blocks import DatasetStoryBlock
-from .forms import BundleAdminForm
-from .panels import BundleNotePanel
-
-
-class BundleStatus(models.TextChoices):
-    PENDING = "PENDING", "Pending"
-    IN_REVIEW = "IN_REVIEW", "In Review"
-    APPROVED = "APPROVED", "Approved"
-    RELEASED = "RELEASED", "Released"
-
+from ons_alpha.datasets.blocks import DatasetStoryBlock
 
-ACTIVE_BUNDLE_STATUSES = [BundleStatus.PENDING, BundleStatus.IN_REVIEW, BundleStatus.APPROVED]
-EDITABLE_BUNDLE_STATUSES = [BundleStatus.PENDING, BundleStatus.IN_REVIEW]
+from .enums import ACTIVE_BUNDLE_STATUSES, EDITABLE_BUNDLE_STATUSES, BundleStatus
+from .forms import BundleAdminForm
+from .panels import BundleNotePanel, PageChooserWithStatusPanel
 
 
 class BundlePage(Orderable):
     parent = ParentalKey("Bundle", related_name="bundled_pages", on_delete=models.CASCADE)
     page = models.ForeignKey("wagtailcore.Page", blank=True, null=True, on_delete=models.SET_NULL)
 
     panels = [
-        PageChooserPanel("page", ["articles.ArticlePage", "bulletins.BulletinPage", "methodologies.MethodologyPage"]),
+        PageChooserWithStatusPanel(
```

Comment:
Non-blocker, can you filter our live pages out of the box?

Reply:
This could be done via `construct_page_chooser_queryset` or better, by registering a custom chooser.

Thinking of the logic for this - we will want to filter out live pages that do not have any pending drafts

Reply:
will look at tackling separately